api/HTMLVideoElement.json

```diff
@@ -49,15 +49,39 @@
       },
       "autoPictureInPicture": {
         "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLVideoElement/autoPictureInPicture",
```

Comment:
This does not seem to be supported yet:
https://mdn-bcd-collector.appspot.com/tests/api/HTMLVideoElement/autoPictureInPicture
https://www.chromestatus.com/feature/5317876315586560